generator-core/src/main/java/br/com/generator/core/GeneratorExecutor.java

```diff
@@ -35,7 +35,7 @@ public File addDependecies(GeneratorOptions options) throws IOException {
         File fileGenerated = new File(options.getDestination());
         FileUtils.writeStringToFile(fileGenerated, contentReplaced);
 
-        System.out.println("Add dependencies in ".concat(options.getDestination()));
+        System.out.println("Add dependencies in ".concat(options.getName()));
         return fileGenerated;
```

Comment:
Ensure `bufferWritter` and `fileWritter` are properly checked for null before calling close to avoid potential NullPointerException.